### PR TITLE
gitqlient: Update to v1.6.2

### PR DESCRIPTION
**Pull request by cyrusbuilt**

### CODE REVIEW

**Comment by korli** [comment]
> I just noticed at runtime: when you click "push", it tries to invoke 'ssh_askpass'. I see there is an existing package for that. Should I add that to the REQUIRES? ie. cmd:ssh_askpass?

No, it looks like an optional dependency.

File: dev-vcs/gitqlient/gitqlient-1.6.2.recipe [hunk 1]
```diff
@@ -20,35 +20,19 @@ suposed to be."
...
 ADDITIONAL_FILES="gitqlient.rdef.in"
-PATCHES="gitqlient-1.2.0.patchset"
 
 ARCHITECTURES="all !x86_gcc2 ?arm ?ppc"
 SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
 	gitqlient$secondaryArchSuffix = $portVersion
-	app:GitQlient
+	app:GitQlient$secondaryArchSuffix = $portVersion
```
Comment (korli) [inline dev-vcs/gitqlient/gitqlient-1.6.2.recipe:35]:
```suggestion
	app:GitQlient= $portVersion
```

File: dev-vcs/gitqlient/gitqlient-1.6.2.recipe [hunk 2]
```diff
@@ -58,13 +42,15 @@ REQUIRES="
 	lib:libQt5Core$secondaryArchSuffix
 	lib:libQt5Gui$secondaryArchSuffix
 	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
```
Comment (korli) [inline dev-vcs/gitqlient/gitqlient-1.6.2.recipe:45]:
```suggestion
	lib:libQt5Gui$secondaryArchSuffix
	lib:libQt5Network$secondaryArchSuffix
	lib:libQt5Widgets$secondaryArchSuffix
```

File: dev-vcs/gitqlient/gitqlient-1.6.2.recipe [hunk 3]
```diff
@@ -58,13 +42,15 @@ REQUIRES="
...
 	"
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	devel:libQt5Core$secondaryArchSuffix
 	devel:libQt5Gui$secondaryArchSuffix
 	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
```
Comment (korli) [inline dev-vcs/gitqlient/gitqlient-1.6.2.recipe:53]:
```suggestion
	devel:libQt5Gui$secondaryArchSuffix
	devel:libQt5Network$secondaryArchSuffix
	devel:libQt5Widgets$secondaryArchSuffix
```

File: dev-vcs/gitqlient/gitqlient-1.6.2.recipe [hunk 4]
```diff
@@ -75,34 +61,29 @@ BUILD_PREREQUIRES="
 	cmd:qmake$secondaryArchSuffix >= 5
 	"
 
-defineDebugInfoPackage gitqlient$secondaryArchSuffix \
```
Comment (korli) [inline dev-vcs/gitqlient/gitqlient-1.6.2.recipe:78]: Any reason to remove?

**Comment by korli** [comment]
Please check and fix the file permissions in this PR: 100644 → 100755

File: dev-vcs/gitqlient/gitqlient-1.6.2.recipe (hunk 1 above)
Comment (cyrusbuilt) [reply dev-vcs/gitqlient/gitqlient-1.6.2.recipe:35]: Fixed

File: dev-vcs/gitqlient/gitqlient-1.6.2.recipe (hunk 2 above)
Comment (cyrusbuilt) [reply dev-vcs/gitqlient/gitqlient-1.6.2.recipe:45]: Fixed. Out of curiosity, does this order matter?

File: dev-vcs/gitqlient/gitqlient-1.6.2.recipe (hunk 3 above)
Comment (cyrusbuilt) [reply dev-vcs/gitqlient/gitqlient-1.6.2.recipe:53]: Fixed

File: dev-vcs/gitqlient/gitqlient-1.6.2.recipe (hunk 4 above)
Comment (cyrusbuilt) [reply dev-vcs/gitqlient/gitqlient-1.6.2.recipe:78]: Not a good one, lol. I removed it when I was troubleshooting packaging errors and I felt like it wasn't entirely necessary. Fixed.

**Comment by Begasus** [comment]
LGTM, @korli fine here?

**Pull request merged**